DOCKER.md

```diff
@@ -48,6 +48,19 @@ By default wait until mysql database is operative at maximum of default DBTIMEOU
     - DBTIMEOUT=100
 ```
 
+
+
+Additionally, the following environment variables are available for keypass docker
+
+| Environment variable        | Configuration attribute   | Default value             |
+|:----------------------------|:--------------------------|:--------------------------|
+| DB_HOST_VALUE               |                           | localhost:3306            |
```

Comment:
Maybe the should use the KEYPASS_ prefix? E.g. KEYPASS_DB_HOST_VALUE

Reply:
fixed in https://github.com/telefonicaid/fiware-keypass/pull/128/commits/a60881ad7ac4c77f7d820677e149acab30e50752